cmd/apply.go

```diff
@@ -27,13 +27,20 @@ import (
 // applyCmd represents the apply command
 var applyCmd = &cobra.Command{
 	Use:   "apply",
-	Short: "A brief description of your command",
-	Long: `A longer description that spans multiple lines and likely contains examples
-and usage of using your command. For example:
+	Short: "Apply desired template file to Sendgrid ",
+	Long: `Given structure of an email:
```

Comment:
Podría ser `of a Sendgrid transactional template` en vez de `email`?

Reply:
yes